packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -151,39 +149,40 @@ export function viewFromState(
 		}) as unknown as FuzzView);
 	return view;
 }
-function filterFuzzNodeSchemas<TreeNodeSchemaIdentifier, FlexTreeNodeSchema>(
-	nodeSchemas: ReadonlyMap<TreeNodeSchemaIdentifier, FlexTreeNodeSchema>,
+function filterFuzzNodeSchemas(
+	nodeSchemas: Iterable<TreeNodeSchemaIdentifier>,
 	prefix: string,
 	omitInitialNodeSchemas: string[],
-): FlexTreeNodeSchema[] {
-	const values: FlexTreeNodeSchema[] = [];
+): TreeNodeSchemaIdentifier[] {
+	const values: TreeNodeSchemaIdentifier[] = [];
 
-	nodeSchemas.forEach((value, key) => {
+	for (const key of nodeSchemas) {
```

Comment:
Nit: should we just call this `schema` or `nodeSchema`? The below `values.push(key)` call is strange :)

Reply:
its not a schema any more, its just the identifier. This function no longer touches any schema.

Reply:
"identifier" then?